google/ggrc-core
# GGRC-8134 / 8135 [MLV] Assessment verifiers

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] **TMP** commit is dropped


# Must be merged into _GGRC-6060-multi-verify_ branch

# Steps to test the changes

1. Open Assessment info-pane (MLV flow should be turned ON for assessment)
2. Check verifiers on info-pane
3. Open Edit modal
4. Check verifiers on edit modal

# Sanity checklist




# PR Review checklist
